src/Components/SavedSearchAlert/SavedSearchAlertModal.tsx

```diff
@@ -178,15 +197,47 @@ export const SavedSearchAlertModal: FC<SavedSearchAlertFormProps> = ({
                   />
                 </Box>
 
-                <Spacer y={2} />
-
                 {values.push && (
-                  <FrequenceRadioButtons
-                    defaultFrequence={values.frequency}
-                    onSelect={selectedOption =>
-                      setFieldValue("frequency", selectedOption)
-                    }
-                  />
+                  <>
+                    <Spacer y={2} />
+                    <FrequenceRadioButtons
+                      defaultFrequence={values.frequency}
+                      onSelect={selectedOption =>
+                        setFieldValue("frequency", selectedOption)
+                      }
+                    />
+                  </>
+                )}
+
+                <Spacer y={4} />
+
+                {isHearFromArtsyAdvisorEnabled && (
+                  <>
+                    <Box display="flex" justifyContent="space-between">
+                      <Text variant="sm-display">
+                        Hear from an Artsy Advisor
+                      </Text>
+                      <Checkbox
+                        onSelect={selected => {
+                          setFieldValue("hearFromArtsyAdvisor", selected)
+                        }}
+                        selected={values.hearFromArtsyAdvisor}
+                      />
+                    </Box>
+
+                    {values.hearFromArtsyAdvisor && (
```

Comment:
Leaving out the phone number stuff for now.

This may _even_ be ok to start rolling out _without_ the phone number - as it may still generate a valid email to advisory with the collector's information + email address - and the phone number could be an addition!